docs/source/ultimate.rst

```diff
@@ -125,6 +125,25 @@ Conserveremo i suoi dati sensibili (nome e cognome, informazioni per contattarla
 
+Polish
+~~~~~~
+(translation courtesy of Karolina Finc, Monika Boruta-Żywiczyńska, and Marcin Koculak)
+
+Dane i próbki zgromadzone w tym badaniu mogą zostać wykorzystane także w innych, przyszłych projektach badawczych, związanych lub niezwiązanych z celami niniejszego badania. Zgromadzone dane, łącznie z danymi z obrazowania mózgu, zostaną zanonimizowane i udostępnione publicznie w ogólnodostępnej internetowej bazie danych. 
+
+Anonimizacja oznacza, że dane, które udostępnimy w internetowej bazie danych, nie będą zawierały Twojego imienia i nazwiska, a jedynie kod uczestnika badania, co uniemożliwi bezpośrednią identyfikację Twojej osoby. Dzięki temu osoby trzecie nie będą mogły ustalić Twojej tożsamości ani określić, które dane należą do Ciebie. Ponadto, nie będziemy udostępniać żadnych innych informacji, takich jak rysy twarzy czy data wykonania Twojego badania, które naszym zdaniem mogłyby pomóc osobom znającym Cię w odgadnięciu, które spośród zgromadzonych danych dotyczą Ciebie. 
+
+Jeśli zmienisz zdanie i wycofasz zgodę na udział w niniejszym badaniu (aby to zrobić możesz skontaktować się z <imię i nazwisko Kierownika Projektu> na <numer telefonu>), nie będziemy gromadzić żadnych dodatkowych danych o Tobie. Usuniemy Twoje dane, jeśli wycofasz się, zanim umieścimy je w internetowej bazie danych. **Niemniej jednak dane oraz wyniki badania, które zostały już udostępnione innym za pośrednictwem internetowej bazy danych lub wykorzystane w publikacji w czasopiśmie naukowym, nie mogą zostać zniszczone, wycofane ani usunięte.**
+
+Twój udział w badaniu to bezinteresowne i hojne wspracie badań naukowców z całego świata, które mogą pomóc innym. Możliwe, że niektóre badania przeprowadzone przy użyciu Twoich danych przyczynią się do opracowania nowych metod badania mózgu, nowych testów diagnostycznych, nowych leków lub innych produktów komercyjnych. Jeśli tak się stanie, nie będziemy mogli zapewnić Ci prawa własności do takich produktów ani jakiejkolwiek części z zysków przez nie wygenerowanych.
+
+Zgodnie z naszą wiedzą, tak udostępniane ogółowi społeczeństwa dane nie będą zawierały żadnych informacji pozwalających na Twoją identyfikację. Twoje dane zostaną zanonimizowane, jednak dzięki wykorzystaniu danych powiązanych z Twoim imieniem w innych rejestrach (na przykład skanów mózgu uzyskanych z Twojej dokumentacji medycznej) istnieje możliwość powiązania Twoich dany obrazowych lub innych informacji z naszej bazy, z powrotem z Tobą. Ponadto, naruszenie zabezpieczeń (włamanie lub cyberatak) może doprowadzić do tego, że ktoś będzie w stanie zidentyfikować Twoje dane. Ryzyko wystąpienia takich zdarzeń jest jednak bardzo niskie, ponieważ Twoje dane są przechowywane w bezpiecznej bazie danych, a informacje wiążące Twoją tożsamość z nadanym kodem uczestnika badania znajdą się w innym rejestrze.
```

Comment:
I liked the previous version "z Twoja osobą", so the sentence looks like:
`...powiązania Twoich dany obrazowych lub innych informacji z naszej bazy z Twoją osobą"